.github/workflows/main.yml

```diff
@@ -186,6 +186,7 @@ jobs:
       - run: cargo doc --no-deps --exclude=risc0-zkvm-methods --workspace
 
   check-benchmarks:
+    if: false # Currently broken. See https://github.com/risc0/risc0/issues/1013
```

Comment:
I think it's OK to still run this and have it fail because currently this status check isn't required to allow PRs to land.

Reply:
Removed this change in 6b167135198f97d7b4ad829d00c699e6f026ab16